README.md

```diff
@@ -1,2 +1,560 @@
-# llrp-device-service
-Device Service for LLRP Messages from 3rd Party RFID Readers
+# Device RFID LLRP Go
+EdgeX device service for communicating with LLRP-based RFID readers.
+
+## Table of contents
+
+* [First Run](#first-run)
+* [Device Discovery](#device-discovery)
+* [Device Profiles, Custom LLRP Messages, and Service Limitations](#device-profiles-custom-llrp-messages-and-service-limitations)
+* [Connection Management](#connection-management)
+* [Example Scripts](#example-scripts)
+* [Testing](#testing)
+* [Snap Development and Testing](#snap-development-and-testing)
+* [Footnotes](#footnotes)
+
+## First Run
+**Build Native**
+```bash
+make build
+```
+**Build Docker**
+```bash
+make docker
+```
+**Configure subnet information**
+```bash
+./bin/auto-configure.sh
```

Comment:
It can be safely run, but the checks will throw an error. We may have skipped the deploy step in these instructions.

https://github.com/edgexfoundry-holding/device-rfid-llrp-go/blob/d5846be48970d81bb51fe0a1c1a04a9b377c15b7/bin/auto-configure.sh#L30-L34

Reply:
I still think a short note mentioning that the service must be running before the script is run would be helpful here.